src/settings/mod.rs

```diff
@@ -122,8 +126,9 @@ impl Settings {
 
         match config.try_into::<Settings>() {
             Ok(settings) => {
-                // TODO: replace this with proper logging when we have it
-                println!("config: {:?}", settings);
+                let logger =
+                    MozlogLogger::new(&settings).expect("Unable to create MozlogLogger instance.");
+                slog_info!(logger, "{}", "Finished configuration successfully."; "settings" => &settings);
```

Comment:
Do we need to include a message where you have `Finished configuration successfully`? Seems like it's just log noise otherwise and this line is already going to be pretty long, so we might thank ourselves in the future if we ditch it or make it as short as possible.

Reply:
Yes, I agree :) I'll remove that.